plato/cli.py

```diff
@@ -54,4 +55,9 @@ def refresh_local_templates():
         """
         Refresh local templates by loading the templates from AWS S3 Bucket
         """
-        load_templates(S3_BUCKET, TEMPLATE_DIRECTORY, S3_TEMPLATE_DIR)
+        file_storage = initialize_file_storage(STORAGE_TYPE)
+        with app.app_context():
+            if STORAGE_TYPE == StorageType.S3:
+                file_storage.load_templates(TEMPLATE_DIRECTORY, TEMPLATE_DIRECTORY_NAME)
```

Comment:
Same here if you apply my suggestion regarding the implementation of load_templates on the subclasses.